/*
 * This file was automatically generated by EvoSuite
 * Sun Oct 02 22:25:58 KST 2016
 */


import static org.evosuite.runtime.EvoAssertions.assertThrownBy;
import static org.junit.Assert.fail;

import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(useVNET = true) 
public class Tool_ESTest5   {

  @Test
  public void test0()  throws Throwable  {
      Tool tool0 = new Tool((TankManager) null, 362, 362, 362);
      // Undeclared exception!
      try { 
        tool0.treffenErkennen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("Tool$$Tools", e);
      }
  }

  @Test
  public void test5()  throws Throwable  {
      Tool tool0 = new Tool((TankManager) null, 370, 370, 370);
      tool0.objectType = 370;
      // Undeclared exception!
      try { 
        tool0.toolMalen();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("Tool", e);
      }
  }

  @Test
  public void test8()  throws Throwable  {
      Tool tool0 = new Tool((TankManager) null, (-611), (-611), (-611));
      // Undeclared exception!
      try { 
        tool0.remove();
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("Tool$$Tools", e);
      }
  }
}
